src/vc/VerifiableCredential.ts

```diff
@@ -134,11 +297,85 @@ export class VerifiableCredential {
                 issuanceDate: this.issuanceDate,
                 identifier: this.identifier,
                 expirationDate: this.expirationDate,
-                version: this.version,
+                // version: this.version,
                 type: this.type,
                 credentialSubject: this.credentialSubject,
             }
         }
+    }
+
+    isMatch = (constraints: any) => {
```

Comment:
I think we should stay consistant with our types of functions? What do you think?

Reply:
Sorry I totally misunderstood you - ignore previous comment.

Agreed, fixing